src/CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ/MassTransitRabbitMQHostingExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Aspire.Hosting.ApplicationModel;
+using CommunityToolkit.Aspire.Hosting.MassTransit.RabbitMQ;
+using Microsoft.Extensions.Configuration;
+
+namespace Aspire.Hosting;
+    
+/// <summary>
+/// MassTransitHostingExtensions provides extension methods for configuring MassTransit in an Aspire-based application.
+/// </summary>
+public static class MassTransitRabbitMqHostingExtensions
```

Comment:
I do not see a reason for having hosting integration for MassTransit; it's not hosting.

We should use the connection string provided from normal `AddRabbitMQ("rabbitmq")` for creating client integration.

Reply:
That's a great point. Having just a client integration makes more sense for this.

Reply:
Agreed - looking at the implementation of the integration, it is wrapping the RabbitMQ integration and doesn't seem to add any additional functionality.